### PR TITLE
Chiyavan

### PR DESCRIPTION
**Pull request by cbanavi**
Finished Week 3 and Week 4

### CODE REVIEW

File: src/chiyavan/week3/PrimeNumber_Chiyavan.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    }
+
+    public static boolean primeNumberChecker(int n) {
+
+        if (n <= 1) {
+            throw new RuntimeException("Prime numbers are greater than 1");
+        }
+
+        for (int i = 2; i * i <= n; i++) {
```
Comment (bernasantiago) [inline src/chiyavan/week3/PrimeNumber_Chiyavan.java:17]: you can exclude last index n, otherwise the code will give wrong answer.

Comment (bernasantiago) [inline src/chiyavan/week3/PrimeNumber_Chiyavan.java:17]: you can exclude the last index of n, otherwise it will give the wrong output.

**Review by bernasantiago — APPROVED** [review]

File: src/chiyavan/week4/SameLetters_Chiyavan.java
Comment (PendarShams) [inline src/chiyavan/week4/SameLetters_Chiyavan.java:1]: lgtm

File: src/chiyavan/week4/FrequencyOfCharacters_Chiyavan.java
Comment (PendarShams) [inline src/chiyavan/week4/FrequencyOfCharacters_Chiyavan.java:1]: lgtm

File: src/chiyavan/week4/RemoveDuplicates_Chiyavan.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package chiyavan.week4;
+
+import java.util.ArrayList;
+
+
+public class RemoveDuplicates_Chiyavan {
+
+    public static void main(String[] args) {
+
+        duplicateChecker("AAABBBCCC");
```
Comment (PendarShams) [inline src/chiyavan/week4/RemoveDuplicates_Chiyavan.java:10]: I believe you have to print you boolean . 
your code is perfect.

**Pull request merged**